### PR TITLE
fix: first balance load for multisig

### PR DESCRIPTION
**Pull request by Asmadek**

### CODE REVIEW

File: src/renderer/screens/Assets/Assets.tsx [hunk 1]
```diff
@@ -58,18 +58,22 @@ export const Assets = () => {
...
+  ).length;
+
   const sortedChains = useMemo(
     () =>
       sortChains(
         Object.values(connections).filter((c) => {
           const isDisabled = c.connection.connectionType === ConnectionType.DISABLED;
           const hasMultisigAccount = activeAccounts.some(isMultisig);
-          const hasMultiPallet = !hasMultisigAccount || Boolean(c.api?.tx.multisig);
+          const hasMultiPallet = !hasMultisigAccount || c.connection.hasMultisigPallet !== false;
```
Comment (tuul-wq) [inline src/renderer/screens/Assets/Assets.tsx:71]: `c.connection.hasMultisigPallet !== false;` should be `true` or `undefined`?

File: src/renderer/domain/connection.ts [hunk 2]
```diff
@@ -9,6 +9,7 @@ export type Connection = {
   customNodes?: RpcNode[];
   activeNode?: RpcNode;
   disconnect?: () => void;
+  hasMultisigPallet?: boolean;
```
Comment (tuul-wq) [inline src/renderer/domain/connection.ts:12]: Does It need to be optional?
I see `hasMultisigPallet: Boolean(api?.tx.multisig)` down below, looks like we have the value all the time

**Review by tuul-wq — COMMENTED** [review]
Some comments

File: src/renderer/screens/Assets/Assets.tsx (hunk 1 above)
Comment (Asmadek) [reply src/renderer/screens/Assets/Assets.tsx:71]: right, it is for the first loading

File: src/renderer/domain/connection.ts (hunk 2 above)
Comment (Asmadek) [reply src/renderer/domain/connection.ts:12]: Not all the time, we have some time between first app start and first chain connection

Same for when we will add new chains

**Review by tuul-wq — APPROVED** [review]

**Pull request merged**